joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstCreatorHelper.scala

```diff
@@ -36,6 +86,10 @@ trait AstCreatorHelper { this: AstCreator =>
     usedVariableNames.put(variableName, counter)
     currentVariableName
   }
+
+  protected def addReceiverEdge(dstNode: nodes.NewNode, srcNode: nodes.NewNode): Unit = {
```

Comment:
Something like this exists in the `Ast()` class already

Reply:
Will remove this code as anyways we are not using this function anywhere in ruby